src/api/ctid-ng.js

```diff
@@ -10,10 +10,18 @@ export default (client: ApiRequester, baseUrl: string) => ({
     return client.put(`${baseUrl}/users/me/presences`, { presence }, token, ApiRequester.successResponseParser);
   },
 
-  listMessages(token: Token, participantUuid: ?UUID): Promise<Array<ChatMessage>> {
-    const body = participantUuid ? { participant_user_uuid: participantUuid } : null;
+  listMessages(token: Token, participantUuid: ?UUID, limit?: number): Promise<Array<ChatMessage>> {
+      const body: Object = {};
```

Comment:
It's not technically a `body`, but more about `query parameters`. A `@GET` request does not even have a body.